tests/dummy/app/pods/numeric-field/controller.js

```diff
@@ -0,0 +1,13 @@
+import Ember from 'ember'
+const {Controller} = Ember
+
+export default Controller.extend({
+  queryParams: ['value'],
+  value: '3',
```

Comment:
Sorry, I just noticed this - the value for a numeric field should really be in a numeric format - is the field capable of handling something other than a string (a number)?